test/with-other-redis-client.test.js

```diff
@@ -0,0 +1,643 @@
+/**
+ * Module dependencies
+ */
+
+var util = require('util');
+var assert = require('assert');
+var _ = require('lodash');
+var async = require('async');
+var Sails = require('sails').Sails;
+var SocketIORedisAdapter = require('socket.io-redis');
+
+// TODO:
+// figure out how to make this run on Travis
+// (need a local redis)
+
+
+describe('with another redis client', function (){
+
+  // Common app config
+  var appConfig = {
+    log: { level: 'warn' },
+
+    globals: false,
+
+    hooks: {
+      // Inject the sockets hook in this repo into this Sails app
+      sockets: require('../')
+    },
+
+    loadHooks: ['moduleloader', 'userconfig', 'http', 'sockets'],
+
+    // Configure the socket.io-redis adapter
```

Comment:
I'd like to extrapolate these tests in some way to prevent duplication, but I can help with that later.  But just want to check first-- is this identical to the existing tests other than the change to the redis client library in the config?